x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -828,6 +826,28 @@ public static void renderDocs() throws IOException {
         FunctionDefinition definition = definition(name);
         if (definition != null) {
             EsqlFunctionRegistry.FunctionDescription description = EsqlFunctionRegistry.description(definition);
+            if (name.equals("case")) {
```

Comment:
nit: Should we use a constant for `case` here and in `RailRoadDiagram.java`? Maybe also in the function registry?

Reply:
Meh.